ui/app/helpers/utils/i18n-helper.js

```diff
@@ -1,19 +1,26 @@
 // cross-browser connection to extension i18n API
 const log = require('loglevel')
 
+const warned = {}
 /**
  * Returns a localized message for the given key
  * @param {object} locale The locale
  * @param {string} key The message key
  * @param {string[]} substitutions A list of message substitution replacements
  * @return {null|string} The localized message
  */
-const getMessage = (locale, key, substitutions) => {
+const getMessage = (currentLocale, locale, key, substitutions) => {
```

Comment:
Could we use `locale` in the map instead of the `currentLocale`? It should have the same effect seeing as whenever a user is getting messages `locale` is going to be the current locale, no?

Reply:
I'm not sure what you mean - what would we use for the key? The `locale` variable should probably be renamed `localeMessages` - it's the map of all of the messages for the current locale. The _name_ or code of the current locale isn't there.

Reply:
I did notice that I forgot to update the JSDoc comment though. I've updated that, and renamed the parameters to be a bit more clear.